NineChronicles.DataProvider.Tests/AgentQueryTest.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using GraphQL.Execution;
+using Libplanet;
+using Libplanet.Crypto;
+using NineChronicles.DataProvider.Store.Models;
+using Xunit;
+
+namespace NineChronicles.DataProvider.Tests;
+
+public class AgentQueryTest : TestBase, IDisposable
+{
+    [Theory]
+    [InlineData(0)]
+    [InlineData(1)]
+    public async Task ShopEquipmentCount(int expected)
```

Comment:
Should this method name be `AgentCount` instead?

Reply:
oops, my mistake. 😅